movies.js

```diff
@@ -17,6 +17,9 @@ export async function loadMovies() {
 export async function loadMovie(id) {
   const res = await fetch(API_BASE + "/movies/" + id);
   const payload = await res.json();
+  if (payload.data == null) {
+    return null;

```

Comment:
This is a fine solution, for `loadMovie()` to return `null` when the movie could not be found. You could also look at `res.status` to figure out what status code you received from the CMS API.